doc/README.md

```diff
@@ -5,11 +5,11 @@ The following diagram shows the overall architecture of the PSENscan software.
 ## Detailed Description
 The overall architecture should give a good idea of how the driver is supposed to work.
 For more details we recommend taking a look at the source code itself:
-* For the **ROS** part we refer to psen_scan_v2::ROSScannerNodeT
-* The referenced **User API** is mainly defined by psen_scan_v2_standalone::api::ScannerV2.
-* For the **Scanner protocol Layer** please check psen_scan_v2_standalone::protocol_layer::ScannerProtocolDef or the ScannerStateMachine.
-* Data **Serialization** and **Deserialization** is for example handled in psen_scan_v2_standalone::data_conversion_layer::monitoring_frame::deserialize() or psen_scan_v2_standalone::data_conversion_layer::scanner_reply::serialize()
-* For the  **Primitive communication layer** we recommend taking a look at psen_scan_v2_standalone::communication_layer::UdpClientImpl
+* For the **ROS** part we refer to `psen_scan_v2::ROSScannerNodeT`.
+* The referenced **User API** is implemented by `psen_scan_v2_standalone::api::ScannerV2`.
+* For the **Scanner protocol layer** please check `psen_scan_v2_standalone::protocol_layer::ScannerStateMachine`.
+* Data **Serialization** and **Deserialization** is for example handled in `psen_scan_v2_standalone::data_conversion_layer::monitoring_frame::deserialize()` or `psen_scan_v2_standalone::data_conversion_layer::scanner_reply::serialize()`.
```

Comment:
namespace is "data_conversion_layer" <=> graph names it "data serializer / deserializer layer"